src/core/tests/library/LegacyEnvLayerExtensions.py

```diff
@@ -1448,6 +1452,18 @@ def run_command_output(self, cmd, no_output=False, chk_err=True):
                                  "\n" + \
                                  "Total download size: 231 k\n" + \
                                  "Operation aborted.\n"
+            elif self.legacy_test_type == 'RevertToImageDefault':
+                if self.legacy_package_manager_name is Constants.YUM:
+                    if cmd.find("systemctl list-unit-files --type=service | grep yum-cron.service") > -1:
+                        code = 0
+                        output = 'Auto update service installed'
+                    elif cmd.find("systemctl list-unit-files --type=service '") > -1:
+                        code = 1
+                        output = 'Auto update service not installed'
```

Comment:
Open question on unexercised code in envlayer

Reply:
Resolved. There was an extra quote in the text within .find() causing a mismatch